index.html

```diff
@@ -128,101 +128,51 @@ <h3>Multi-Post Stories</h3>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
-        <div class="job5">
-          <h2>Profesional Art Printing Data</h2>
-          <div class="info-data">
-            <p>
-              A daily section of daily private personalised reads; no accounts
-              or sign ups required ha been the industry standups
-            </p>
-          </div>
-          <ul class="grid-container2">
-            <li class="item1"><a href="#">html</a></li>
-            <li class="item2"><a href="#">bootstrap</a></li>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
-        <div class="job6">
-          <h2>Profesional Art Printing Data</h2>
-          <div class="info-data">
-            <p>
-              A daily section of daily private personalised reads; no accounts
-              or sign ups required ha been the industry standups
-            </p>
-          </div>
-          <ul class="grid-container2">
-            <li class="item1"><a href="#">html</a></li>
-            <li class="item2"><a href="#">bootstrap</a></li>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
+      <!--modal-->
+      <div class="modal">
+        <div class="poster poster-modal">
+         <div class="post-information modal-info">
+         <div class="post-title modal-posttitle">
+          <h3 class="heading-modal">Multi-Post Stories</h3>
+          <div class="close-modal">&times;</div>
+         </div>
+         <ul class="grid-container" id="modal-list-grid">
+         
+         </ul>
+         <div class="image-holder image-modal">
+          <img class="post-images" src="./assets/post2.svg" alt="Post" />
+         </div>
+        </div>   
+     <div class="post post-modal">
+      <p class="paragraph"> 
+        A daily selection of private personalised reads; o accounts or
+            signups required. Has been the industry's dummy standard text ever
+            snce the 1500s. When an unknown printer took a standard dummy text
+      </p>
+    </div>
+    <!--
+     <div class="btns">
+      <div class="btns-container">
+      <span class="working-btns">See Live</span>
+      <img class="icons" src="./assets/github.png" alt="">
+    </div>
+    <div class="btns-container">
+      <span class="working-btns">See Live</span>
+      <img class="icons" src="./assets/backto.png" alt=""/>
+    </div>
+     </div>-->
+     <ul class="btns">
+       <li><span class="btns-container"><a href="">See Live <img class="icons" src="./assets/github.png" alt=""></a></span></li>
+       <li><span class="btns-container"><a href="">See Live <img class="icons" src="./assets/backto.png" alt=""/></a></span></li>
+     </ul>
+   </div>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly make sure that `"See Live"` are buttons which means they are in `<button> tags`. Also, make sure to add the `button interactions (hover, active, pressed)` as per the Figma design.
<img width="376" alt="Screenshot 2022-04-07 at 7 18 18 AM" src="https://user-images.githubusercontent.com/70269507/162119330-73996556-c2c6-4542-ada7-dc3f4b70a313.png">

- As you can see below that your buttons are not having button interactions. Kindly fix it.

https://user-images.githubusercontent.com/70269507/162119316-7a60cdf3-a56b-42ed-91a8-6e3152bce57c.mov